index.html

```diff
@@ -148,6 +148,35 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        <!-- ==================  TEMPLATE  ================== -->
```

Comment:
Looks like you accidentally copied too many lines when you were making a copy of the template! 😅 

No worries, it happens to the best of us. Just remove this line and the one below it and you'll be set. 💪

Reply:
I think I have made the change.